src/js/stores/font.js

```diff
@@ -237,7 +234,7 @@ define(function (require, exports, module) {
                 };
             }
 
-            if (textStyle.leading >= 0) {
```

Comment:
This will filter out `null` value in `textStyle.leading`, which is also `null >= 0`